src/libostree/ostree-bootconfig-parser.c

```diff
@@ -56,6 +59,94 @@ ostree_bootconfig_parser_clone (OstreeBootconfigParser *self)
   return parser;
 }
 
+/**
+ * ostree_bootconfig_parser_parse_tries:
+ * @self: Parser
+ * @path: File path
+ *
+ * Parses a suffix of two counters in the form "+LEFT-DONE" from the end of the
+ * filename (excluding file extension).
+ */
+static void
+ostree_bootconfig_parser_parse_tries (OstreeBootconfigParser *self,  const char *filename)
```

Comment:
I think this one would be easier to unit test if it was just
```
parse_bootloader_tries (const char *filename, guint32 *out_left, guint32 *out_done, GError **error)
```

(I also added an error so we can error if we fail to parse the numbers which seems like a good idea?)

Then it'd be good to have some unit tests for that. Actually, I will try to do a prep PR to add unit tests for some other code in this file just to start that ball.